PowerPointLabs/PowerPointLabs/PictureSlidesLab/Service/StylesDesigner.cs

```diff
@@ -41,7 +41,7 @@ public sealed class StylesDesigner : PowerPointPresentation, IStylesDesigner
         public StylesDesigner(Application app = null)
         {
             Path = TempPath.TempFolder;
-            Name = "PictureSlidesLabPreview" + Guid.NewGuid().ToString().Substring(0, 7);
```

Comment:
@damithc

I see. The change was done as the old setter for `Name` appended the `.pptx` extension whenever the Name field is set.

[Relevant line](https://github.com/PowerPointLabs/PowerPointLabs/pull/1201/files#diff-87b0ea049b75914d91a33f9493b97058L76)